docs/src/man/comparisons.md

```diff
@@ -235,6 +235,70 @@ The table below compares more advanced commands:
 | DataFrame as input        | `summarize(df, head(across(), 2))`                        | `combine(d -> first(d, 2), df)`                               |
 | DataFrame as output       | `summarize(df, tibble(value = c(min(x), max(x))))`        | `combine(:x => x -> (value = [minimum(x), maximum(x)],), df)` |
 
+
+## Comparison with the R package data.table
+
+The following table compares the main functions of DataFrames.jl with the R package data.table (version 1.14.1). 
+
+```R
+library(data.table)
+df  <- data.table(grp = rep(1:2, 3), x = 6:1, y = 4:9,
+                  z = c(3:7, NA), id = letters[1:6])
+df2 <- data.table(grp=c(1,3), w = c(10,11))                 
+```
+
+| Operation                          | data.table                                       | DataFrames.jl                                | 
+|:-----------------------------------|:-------------------------------------------------|:---------------------------------------------|
+| Reduce multiple values             | `df[, .(mean(x))]`                               | `combine(df, :x => mean)`                    |
+| Add new columns                    | `df[, x_mean:=mean(x) ]`                         | `transform(df, :x => mean => :x_mean)`       |
```

Comment:
elsewhere the in-place distinction is drawn -- IINM, it should be `transform!` for this row?

Two `data.table` ways to do it not in-place:

```
copy(df)[, x_mean:=mean(x)]
df[ , c(.SD, list(x_mean = mean(x))]
```

Reply:
the intention is that `data.table` and DataFrames.jl operations are doing the same. If `df[, x_mean:=mean(x) ]` is in-place, then we should use `transform!`, additional alternatives are:
* `df.x_mean = mean(df.x)`
* `insertcols!(df, :x_mean => mean(df.x))`

Reply:
yes, `:=` is always in-place

Reply:
Thanks. That was a typo, it should have been `transform!`